app/models/ems_event.rb

```diff
@@ -227,6 +227,7 @@ def get_target(target_type)
 
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
+    target_type = "middleware_server"   if event.event_type == "hawkular_event"
```

Comment:
Do we need a more specific event_type for alert-related events?  We don't want these events to show up on the timeline, right?  So, perhaps we need an event type like hawkular_alerting.hidden or something like that.  We could then filter these 'hidden' events from showing up on the the timeline. 

Also, is "middleware_server" an appropriate target_type?  I'm not sure what the target_type is for but this seems very specific.  Are we going to need a different target_type for various entities we may need to alert on?

We can discuss these things offline if you like, to make sure I'm understanding. And then update the PR with answers.